scripts/build.js

```diff
@@ -12,6 +12,27 @@ const cmake = require("cmake-js");
 const axios = require("axios");
 const tar = require('tar');
 
+function rmRecursive(path) {
+    var files = [];
+    if (fs.existsSync(path)) {
+        if (fs.lstatSync(path).isDirectory()) {
+            files = fs.readdirSync(path);
```

Comment:
Do we need to filter out `.` and `..` so we don't wipe a drive/mount point?

Reply:
I feel it's more like a user's responsibility to not delete the thing they need. Like this function is just a tool, user should be responsible to not use it wipe mount point.

Reply:
changed mind, as yeah, rm doesn't rm . and .. as well